app/controllers/admin/api_controller.rb

```diff
@@ -0,0 +1,54 @@
+class Admin::ApiController < ApplicationController
+  before_filter :admin_only
+  before_filter :check_for_cancel, only: [:create, :update]
+
+  def index
```

Comment:
yes about redis, I agree that's a better place to store the info, and yes, the point is just to have an audit trail -- as long as you're the only one who has the API key I agree it's fine not to have it, but I do think that needs to be in place before we give one to anyone not on the coding team